docs/user-guide/systemrequirements-zosmf-ha.md

```diff
@@ -0,0 +1,110 @@
+# Configuring z/OSMF high availability in Sysplex
+
+z/OSMF HA should be configured in Hot Standby mode to ensure availability of REST services. The goal of this configuration is to ensure that one z/OSMF server is always available to provide the REST services.
+It means that there should be at least one backup (hot-standby) server and a preferred target server. Both targets are active, and both z/OSMF servers are bound to the DVIPA.
+So, preferred z/OSMF server receives all new incoming requests. If it fails or the system becomes down, new requests are routed to the backup (hot-standby) server.
+In this mode, the distributing DVIPA does not perform load balancing of requests across multiple systems.
+
+For complete information, go to IBM Documentation and read the following documents:
+
+- [Configuring z/OSMF for availability](https://www.ibm.com/docs/en/zos/2.2.0?topic=environment-configuring-zosmf-availability)
+- [Configuring z/OSMF for high availability](https://www.ibm.com/docs/en/zos/2.4.0?topic=configurations-configuring-zosmf-high-availability)
+
+## Sysplex environment requirements for z/OSMF REST Services
+Ensure that the Sysplex environment meets the following requirements:
+
+- Shared SAF database, see [Sharing a database with sysplex communication in data sharing mode](https://www.ibm.com/docs/en/zos/2.1.0?topic=sd-sharing-database-sysplex-communication-in-data-sharing-mode)
+
+- USS Shared file system, see [How to share file systems in a Sysplex](https://www.ibm.com/docs/en/zos/2.4.0?topic=planning-sharing-file-systems-in-sysplex)
+
+- JESPlex/JES2 Multi-Access Spool (MAS) environment
+
+- Sysplex distributor, configured Dynamic VIPA TCP/IP address
+
+- Extended MCS console (EMCS)
+
+## Setting up z/OSMF nucleus
+
+**Follow these steps if it is a first-time z/OSMF setup:**
+
+1. Create security configuration by sample JCL **SYS1.SAMPLIB(IZUSEC)**
```

Comment:
Thanks @nannanli for making those changes, looks good. I believe **IZUNUSEC** provides all a user requires to set up z/OSMF for HA. But, we could mention that for a full installation of z/OSMF including the nucleus and core services, user can use **IZUSEC** instead of IZUNUSEC.

Reply:
@nannanli Ashley, just wanted to let you know that we are also working on #1688 for Zowe HA. It is not a complete PR yet.